pyramid_crow/__init__.py

```diff
@@ -1,13 +1,41 @@
 from pyramid.tweens import EXCVIEW
 from pyramid.httpexceptions import WSGIHTTPException
+from pyramid.util import DottedNameResolver
 from raven import Client
 
+resolver = DottedNameResolver(None)
+
+PY3 = sys.version_info[0] == 3
+
+if PY3: # pragma: no cover
+    import builtins
+else:
+    import __builtin__ as builtins
+
+def as_globals_list(value):
+    L = []
+    value = aslist(value)
+    for dottedname in value:
+        if dottedname in builtins.__dict__:
+            if PY3: # pragma: no cover
+                dottedname = 'builtins.%s' % dottedname
+            else:
+                dottedname = '__builtin__.%s' % dottedname
+        obj = resolver.resolve(dottedname)
+        L.append(obj)
+    return L
+
 
 def crow_tween_factory(handler, registry):
+
+    get = registry.settings.get
+
+    ignored = get('pyramid_crow.ignore', (WSGIHTTPException,))
```

Comment:
Would prefer `WSGIHTTPException` being automatically added to the list if it's not already there, as that's kinda a gotcha.